package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
-  "version": "0.1.44",
+  "version": "0.1.45",
```

Comment:
don't change the version... this will be done automatically when we publish the file